### PR TITLE
Add option to reload fstab config with `mount -a`

### PR DESCRIPTION
**Pull request by Mr-Pi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ A running linux system.
 | `fstab_proc`              | boolean                             |                `true`               | Generate a fstab entry for procfs                                                                                |
 | `fstab_proc_options`      | list of strings                     |            `['defaults']`           | procfs mount options                                                                                             |
 | `fstab_additional_mounts` | [list of dicts](#additional-mounts) |                 `[]`                | Add additional mount entries                                                                                     |
+| `fstab_reload_method`     | string                              |                `none`               | Reload method for changes in fstab, valid values are `none`(just update fstab), `reload`(performe `mount -a`)    |
```
Comment (SlothOfAnarchy) [inline README.md]:
```suggestion
| `fstab_reload_method`     | string                              |                `none`               | Reload method for changes in fstab, valid values are `none`(just update fstab), `reload`(perform `mount -a`)     |
```

File: handlers/main.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+- name: Mount all fstab entries `mount -a`
```
Comment (SlothOfAnarchy) [inline handlers/main.yml]:
```suggestion
- name: Mount all fstab entries using `mount -a`
```

File: handlers/main.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+  shell: mount -a
+  listen: fstab_update#
```
Comment (SlothOfAnarchy) [inline handlers/main.yml]: I don't think this `#` is intended
```suggestion
  listen: fstab_update
```

**Review by SlothOfAnarchy — CHANGES_REQUESTED** [review]
I was about to complain that `fstab_reload_method` is not a boolean. @Mr-Pi told me he will add a reboot option today or tomorrow.

File: README.md [hunk 4]
```diff
@@ -25,6 +25,7 @@ A running linux system.
 | `fstab_proc`              | boolean                             |                `true`               | Generate a fstab entry for procfs                                                                                |
 | `fstab_proc_options`      | list of strings                     |            `['defaults']`           | procfs mount options                                                                                             |
 | `fstab_additional_mounts` | [list of dicts](#additional-mounts) |                 `[]`                | Add additional mount entries                                                                                     |
+| `fstab_reload_method`     | string                              |                `none`               | Reload method for changes in fstab, valid values are `none`(just update fstab), `reload`(perform `mount -a`)     |
```
Comment (haslersn) [inline README.md]: `reload` or `remount`?

**Review by haslersn — DISMISSED** [review]

File: README.md [hunk 5]
```diff
@@ -12,19 +12,21 @@ A running linux system.
...
+| `fstab_proc`              | boolean                             | `true`                              | Generate a fstab entry for procfs                                                                                |
+| `fstab_proc_options`      | list of strings                     | `['defaults']`                      | procfs mount options                                                                                             |
+| `fstab_additional_mounts` | [list of dicts](#additional-mounts) | `[]`                                | Add additional mount entries                                                                                     |
+| `fstab_reload`            | boolean                             | `false`                             | Reload config after changes                                                                                      |
+| `fstab_reload_method`     | string                              | `remount`                           | Reload method for changes in fstab, currently only `reload` is implemented. `reboot` may be added in the future. |
```
Comment (Mr-Pi) [inline README.md]: remount

**Review by SlothOfAnarchy — APPROVED** [review]

**Pull request merged**